### PR TITLE
Error on deprecated cleanup

### PR DESCRIPTION
**Pull request by peter-lawrey**

### CODE REVIEW

**Review by tgd — APPROVED** [review]

File: src/main/java/net/openhft/chronicle/core/util/ByteBuffers.java [hunk 1]
```diff
@@ -27,10 +27,7 @@
  * Utility class for working with {@link ByteBuffer} instances.
  * <p>
  * Provides functionality to directly set the address and capacity of a ByteBuffer.
- * <p>
- * Note: This class is deprecated and will be removed in version x.26.
  */
-@Deprecated(/* to be removed x.26 */)
```
Comment (JerryShea) [inline src/main/java/net/openhft/chronicle/core/util/ByteBuffers.java]: why has this been undeprecated?

File: src/main/java/net/openhft/chronicle/core/util/Ints.java [hunk 2]
```diff
@@ -32,7 +32,6 @@
  * A collection of functional compositions to check and assert various requirements
  * related to {@code int} values.
  */
-@Deprecated(/* to be moved in x.26 */)
```
Comment (JerryShea) [inline src/main/java/net/openhft/chronicle/core/util/Ints.java:35]: why move deprecated tags to functions?

File: src/main/java/net/openhft/chronicle/core/util/Longs.java [hunk 3]
```diff
@@ -68,6 +61,28 @@ public static long requireNonNegative(final long value) {
...
+     * public Foo(long bar) {
+     *     this.bar = requirePositive(bar);
+     * }
+     * </pre></blockquote>
+     *
+     * @param value the value to check
+     * @return the provided {@code value} if the check passes
+     * @throws IllegalArgumentException If the check fails
+     */
+    public static long requirePositive(final long value) {
```
Comment (JerryShea) [inline src/main/java/net/openhft/chronicle/core/util/Longs.java:80]: some of these are being kept?

File: src/main/java/net/openhft/chronicle/core/util/Ints.java (hunk 2 above)
Comment (peter-lawrey) [reply src/main/java/net/openhft/chronicle/core/util/Ints.java:35]: A few functions are used many times, but most aren't used at all.

File: src/main/java/net/openhft/chronicle/core/util/Longs.java (hunk 3 above)
Comment (peter-lawrey) [reply src/main/java/net/openhft/chronicle/core/util/Longs.java:80]: the ones that were used, yes.

File: src/main/java/net/openhft/chronicle/core/util/ByteBuffers.java (hunk 1 above)
Comment (peter-lawrey) [reply src/main/java/net/openhft/chronicle/core/util/ByteBuffers.java]: The class is needed in one place. So I have moved to that place on review.

**Pull request merged**